org.eclipse.xtext.common.types.tests/.classpath

```diff
@@ -43,6 +43,13 @@
 			<attribute name="test" value="true"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry including="**/*.java|**/*.xtend" kind="src" output="bin/test" path="xtend-gen">
```

Comment:
why is this there now?

Reply:
do you have locally uncommited stuff?

Reply:
yes, the xtend-gen folder was still there